src/clj/game/core/engine.clj

```diff
@@ -639,7 +639,7 @@
   Gather all registered handlers from the state, then append the card-abilities if appropriate,
   then filter to remove suppressed handlers and those whose req is false.
   This is essentially Phase 9.3 and 9.6.7a of CR 1.1:
-  http://nisei.net/files/Comprehensive_Rules_1.1.pdf"
+  https://nullsignal.games/rules/comp-rules/"
```

Comment:
If you're gonna replace this, make sure to update exactly where in the doc it's pointing to. I linked 1.1 on purpose to avoid updates changing the location.